cirq/linalg/operator_sum_utils.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utils for the computation of operator sum (Kraus operators)."""
+
+import numpy as np
+
+from cirq.linalg.transformations import targeted_left_multiply
+from cirq import protocols
+
+
+def compute_kraus_operations(initial_density_matrix: np.ndarray, noisy_circuit, qubits):
```

Comment:
Add type annotations to the `noisy_circuit`, `qubits` variables and also the return type "list of Kraus operators"